pkg/console/subresource/deployment/deployment.go

```diff
@@ -238,15 +238,16 @@ func consoleVolumeMounts(vc []volumeConfig) []corev1.VolumeMount {
 
 func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	flag := ""
-	// Since the console-operator logging has different logging levels then the capnslog,
-	// that we use for console server(bridge) we need to map them to each other
+	// console server(bridge) expects flag log-level
 	switch logLevel {
 	case operatorv1.Normal:
-		flag = "--log-level=*=NOTICE"
+		flag = "--log-level=2"
```

Comment:
We will want the `klog` expected `flag` long term, this should be:

```
-v=2
-v=4
-v=6
-v=10
```

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

Nevermind, your #s are good.

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

disregard, these #s are fine.